travis/test_server.py

```diff
@@ -282,6 +282,8 @@ def main(argv=None):
     unbuffer = str2bool(os.environ.get('UNBUFFER', True))
     data_dir = os.environ.get("DATA_DIR", '~/data_dir')
     test_enable = str2bool(os.environ.get('TEST_ENABLE', True))
+    dbtemplate = os.environ.get('MQT_TEMPLATE_DB', 'openerp_template')
```

Comment:
Wouldn't you agree that TEMPLATE_DB is potentially conflicting in any kind of environment? On the other hand the matrix brings a level of isolation. Though say I have a deployment script working with a variable TEMPLATE_DB as well, things can get rapidly pretty confusing. So far the rational, tell me if you nevertheless prefer that I strip it. In my humble opinion, sooner or later we will need this anyhow, as use becomes more widespread.